### PR TITLE
Select theme

### PR DESCRIPTION
**Pull request by smhigley**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Adds theme for Select widget:
screen shot 2017-07-14 at 11 17 08 am

### CODE REVIEW

File: src/select/Select.ts [hunk 1]
```diff
@@ -250,7 +250,7 @@ export default class Select extends SelectBase<SelectProperties> {
 		});
 	}
 
-	renderNativeSelect(): DNode {
+	protected renderNativeSelect(): DNode {
```
Comment (bitpshr) [inline src/select/Select.ts]: I'm all for this change, but I'm not sure we should touch visibility as part of this theme-specific PR.

File: src/select/Select.ts [hunk 2]
```diff
@@ -288,15 +288,15 @@ export default class Select extends SelectBase<SelectProperties> {
 				onfocus: this._onFocus,
 				onkeydown: this._onKeyDown
 			}, optionNodes),
-			v('span', { classes: this.classes(css.nativeArrow) }, [
+			multiple ? null : v('span', { classes: this.classes(css.arrow) }, [
```
Comment (bitpshr) [inline src/select/Select.ts]: Does this need a hidden text label? (I may be confusing this with buttons.)

File: src/themes/dojo/select.m.css [hunk 3]
```diff
@@ -13,53 +15,41 @@
 /* custom input styles */
 .trigger {
 	background-color: transparent;
-	border: 3px solid var(--input-border-color);
+	border: var(--border-width) solid var(--color-border);
+	border-bottom-color: var(--color-border-strong);
 	font-size: inherit;
+	line-height: var(--line-height-base);
 	min-width: 200px;
```
Comment (bitpshr) [inline src/themes/dojo/select.m.css]: Why the min width? Also, are we making all inputs take up 100% of their parent, but giving them a uniform `max-width`?

File: src/select/Select.ts (hunk 2 above)
Comment (smhigley) [inline src/select/Select.ts]: It's a `<select>` element so it's labeled by its `<label>`. The arrow here is just extra visual window dressing :)

File: src/select/Select.ts (hunk 1 above)
Comment (smhigley) [inline src/select/Select.ts]: I think it was already visible, just missing the `protected` keyword. The other similar fn's have it, so I thought I'd sneak it in :P

File: src/themes/dojo/select.m.css (hunk 3 above)
Comment (smhigley) [inline src/themes/dojo/select.m.css]: I forgot, and let it sneak in. Good catch! I believe we're letting all inputs take up 100% of the parent, with no `max-width`

**Pull request merged**